tests/sandbox/create_message_batches/test_invalid_routing_plan.py

```diff
@@ -50,7 +62,19 @@ def test_no_such_routing_plan(nhsd_apim_proxy_url, correlation_id):
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_invalid_routing_plan(nhsd_apim_proxy_url, correlation_id):
     """
-    .. py:function:: Test accessing another users routing plan
+    .. py:function:: Scenario: A user submitting a request with a routing plan \
```

Comment:
`Scenario: A user submitting a request where the routing plan identifier is not a properly formed UUID receives a validation error response`